library/std/src/keyword_docs.rs

```diff
@@ -3,8 +3,8 @@
 /// Cast between types, or rename an import.
```

Comment:
Since these docs actually talk about all three of them:

```suggestion
/// Cast between types, rename an import or qualifying paths to associated items.
```